### DISCUSSION TITLE
We used Histolab to preprocess WSI, we want to quote Histtolab in the literature, I wonder if there is any official article we can quote?

### DISCUSSION THREAD

**Discussion by zqy396** [histolab/histolab#401] [Q&A] [answered]
We used Histolab to preprocess WSI, we want to quote Histtolab in the literature, I wonder if there is any official article we can quote?

**Comment by alessiamarcolini** [comment, marked as answer]
Answered in issue https://github.com/histolab/histolab/issues/400 :)
